test/simulation/xudtest/node.go

```diff
@@ -54,6 +55,8 @@ func (cfg nodeConfig) genArgs() []string {
 	args = append(args, "--loglevel=debug")
 
 	args = append(args, fmt.Sprintf("--xudir=%v", cfg.DataDir))
+	args = append(args, fmt.Sprintf("--dbpath=%v", cfg.DBPath))
```

Comment:
`cfg.DBPath` isn't initialized with a value. Do we need it?

Reply:
Not any longer actually, it's a leftover from using `:memory:`. So I'll just remove this.